tests/storage/test_event_push_actions.py

```diff
@@ -150,7 +175,7 @@ def _mark_read(stream: int, depth: int) -> None:
 
         _assert_counts(1, 0)
 
-        _mark_read(7, 7)
+        _mark_read(6, 6)
```

Comment:
Is this a change in behavior then?

Reply:
No, this is just making sure we're inject a read receipt at an actual event (which we have to), rather than at some arbitrary place.

(Side note: we need to rewrite this test to not poke things directly into the DB, and instead create real rooms, events and receipts)